### DISCUSSION TITLE
Any chance it can run on a Mac?

### DISCUSSION THREAD

**Discussion by ibboaz** [stohrendorf/CroftEngine#280] [General]
Any chance the code can be built on a modern (non-Intel chipset) Mac? I noticed you were using OpenGL - which of course on a Mac is bad/good: bad because OpenGL on Mac is essentially dead - but good because translation layers like ANGLE/MetalANGLE are a good possibility. 

Just curious - and really nice work BTW.

**Comment by stohrendorf** [comment]
As I don't own a mac, it's basically impossible for me to develop for it.
